sources/packages/backend/apps/workers/src/controllers/supporting-user/_test_/e2e/supporting-user.controller.createSupportingUsers.e2e-spec.ts

```diff
@@ -0,0 +1,149 @@
+import { SupportingUserType } from "@sims/sims-db";
+import {
+  createE2EDataSources,
+  E2EDataSources,
+  saveFakeApplication,
+} from "@sims/test-utils";
+import { saveFakeSupportingUser } from "@sims/test-utils/factories/supporting-user";
+import { IOutputVariables } from "zeebe-node";
+import {
+  createFakeWorkerJob,
+  FAKE_WORKER_JOB_RESULT_PROPERTY,
+  MockedZeebeJobResult,
+} from "../../../../../test/utils/worker-job-mock";
+import { createTestingAppModule } from "../../../../testHelpers";
+import { SupportingUserController } from "../../supporting-user.controller";
+import {
+  CreateSupportingUsersJobInDTO,
+  CreateSupportingUsersJobOutDTO,
+} from "../../supporting-user.dto";
+import { createFakeCreateSupportingUsersPayload } from "./create-supporting-users";
+
+describe("SupportingUserController(e2e)-createSupportingUsers", () => {
+  let db: E2EDataSources;
+  let supportingUserController: SupportingUserController;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    db = createE2EDataSources(dataSource);
+    supportingUserController = nestApplication.get(SupportingUserController);
+  });
+
+  it("Should create parent supporting users when requested.", async () => {
```

Comment:
I would change the descriptions to indicate multiple parents: "Should create parents supporting users when requested.".